src/manage_sql.c

```diff
@@ -26967,6 +27199,11 @@ compare_port_severity (gconstpointer arg_one, gconstpointer arg_two)
 void buffer_results_xml (GString *, iterator_t *, task_t, int, int, int, int,
                          int, int, int, const char *, iterator_t *, int);
 
+/** @todo Defined in gmp.c! */
+void buffer_results_xml_cert (GString *, iterator_t *, task_t, int, int, int,
```

Comment:
I'd prefer to have only version of `buffer_results_xml ()`, but if you think updating all the other uses would be too much
for this PR, I'd be fine with having it done later.

Reply:
No problem, done in 55f6cf2.